module/localisation/RobotParticleLocalisation/src/RobotParticleLocalisation.cpp

```diff
@@ -56,6 +58,13 @@ namespace module::localisation {
                 field->position        = position.matrix();
                 field->covariance      = filter.getCovariance();
 
+                if (config.debug) {
```

Comment:
No need for debug configs, just set the modules log level and log with a specific log level